connectors/salesforce/src/main/java/com/vmware/connectors/salesforce/SalesforceController.java

```diff
@@ -614,8 +780,8 @@ private SFAccount setAccOpportunities(
                 .stream()
                 .map(acct ->
                         new Card.Builder()
-                                .setName("Salesforce")
-                                .setTemplate(routingPrefix + "templates/generic.hbs")
+                                .setName(SALESFORCE)
```

Comment:
Since there were some more setName calls added before and in this PR that don't have the `// TODO - remove this in APF-536`, could you add that comment in Card.java (on the field and the setter in the builder -- I just checked and I hadn't added the comment there).

Then when someone works on APF-536, they'll be sure to remove the property and the methods on the builder, which will cause compiler errors on all of the usages (and we won't miss any of these new references).

Reply:
Added comment in ```Card.java```.